inc/saiacl.h

```diff
@@ -816,7 +816,7 @@ typedef enum _sai_acl_table_attr_t
     /**
      * @brief Range type defined
      *
-     * @type sai_acl_range_type_t
+     * @type sai_s32_list_t sai_acl_range_type_t
```

Comment:
discussed with @itaibaz , it seems s32_list is needed by mellanox since they need to know specify range type at the acl table creation time. @AshokDaparthi , please review.